test/lib/jdk/test/lib/Asserts.java

```diff
@@ -285,8 +285,7 @@ public static void assertEqualsByteArray(byte[] lhs, byte[] rhs, String msg) {
     public static void assertNotEqualsByteArray(byte[] lhs, byte[] rhs, String msg) {
         if (Arrays.equals(lhs, rhs)) {
             msg = Objects.toString(msg, "assertNotEqualsByteArray")
-                    + ": expected " + HexFormat.of().formatHex(lhs)
-                    + " to not equal " + HexFormat.of().formatHex(rhs);
+                    + ": expected not equals but was " + HexFormat.of().formatHex(lhs);
```

Comment:
Seems still a bit long and hard to read. How about just ": should not equal to "?

Reply:
I think they have the same meaning, but I prefer mine because it aligns with the style used in other methods. It sounds more formal and explicitly states what was expected and what's the actual result.